catalogbuilder/intakebuilder/getinfo.py

```diff
@@ -108,6 +108,13 @@ def getInfoFromGFDLFilename(filename,dictInfo,logger,configyaml):
             output_file_template = builderconfig.output_file_template
         except:
             sys.exit("No output_path_template found. Check configuration.")
+    if( "static" in filename ):
+        ## For static we handle this differently . The GFDL PP expected pattern is atmos.static.nc
+        #TODO error checking as needed
+        output_file_template = ['realm'] 
+        dictInfo["variable_id"] = "fixed" 
+        dictInfo["frequency"] = "fx"
+        dictInfo["table_id"] = "fx"
```

Comment:
I don't strongly disagree with this, but I disagree a little.

Sometimes this will be `Ofx` instead of `fx` and maybe something else.

Reply:
So true, I missed that. Ofx fx are likely the only ones. 
Let's keep frequency="fx" but table_id="Ofx" if realm has "ocean" in it. What do you think about this?

Reply:
A good compromise, agreed.

Reply:
```
activity_id,institution_id,source_id,experiment_id,frequency,realm,table_id,member_id,grid_label,variable_id,time_range,chunk_freq,platform,dimensions,cell_methods,standard_name,path
dev,,fre,CM4.5v01_om5b06_piC_noBLING,fx,ocean_monthly,Ofx,,,fixed,,,gfdl.ncrc5-intel23-prod-openmp,,,,/archive/a1r/fre/FMS2024.02_OM5_20240724/CM4.5v01_om5b06_piC_noBLING/gfdl.ncrc5-intel23-prod-openmp/pp/ocean_monthly/ocean_monthly.static.nc
dev,,fre,CM4.5v01_om5b06_piC_noBLING,mon,ocean_monthly,,,,zos,000101-000512,5yr,gfdl.ncrc5-intel23-prod-openmp,,ts,sea_surface_height_above_geoid,/archive/a1r/fre/FMS2024.02_OM5_20240724/CM4.5v01_om5b06_piC_noBLING/gfdl.ncrc5-intel23-prod-openmp/pp/ocean_monthly/ts/monthly/5yr/ocean_monthly.000101-000512.zos.nc
```

Reply:
resolved.